[INFO ] Logging to: /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario/log-run23.txt
[INFO ] Version of SMAC is v2.10.03-master-778 (3ee628ef9bf2), running on OpenJDK 64-Bit Server VM (1.8.0_442) and Linux 4.18.0-553.36.1.el8_10.x86_64 (amd64) 
[INFO ] Call String: smac --scenario-file Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/RandomForest_Iris_scenario.txt --seed 23 
[INFO ] SMAC started at: Apr 14, 2025 8:16:55 AM. Minimizing mean quality.
[INFO ] First incumbent: config 1 (internal ID: 0x000B), with mean quality: -1.0; estimate based on 1 runs.
[INFO ] Sample call for new incumbent config 1 (internal ID: 0x000B): 
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 613434 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:13_176509_2449345976.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:13_176509_2449345976.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:13_176509_2449345976.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.962921672496193", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.44880933629471037", "min_samples_leaf": "22", "min_samples_split": "19", "min_weight_fraction_leaf": "0.45798069453017387", "n_estimators": "7", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "613434", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 613434 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.962921672496193' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.44880933629471037' -min_samples_leaf '22' -min_samples_split '19' -min_weight_fraction_leaf '0.45798069453017387' -n_estimators '7'  
[WARN ] Detected CRASHED run. The Solution Quality of CRASHED runs will be transformed to the MAX(quality, 1.0E9), to disable this use --transform-crashed-quality false to change the value use --transform-crashed-quality-value
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:43_176978_3712200801.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:43_176978_3712200801.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:43_176978_3712200801.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.02654431626624354", "min_samples_leaf": "57", "min_samples_split": "65", "min_weight_fraction_leaf": "0.08930549150722683", "n_estimators": "379", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "7828936", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 7828936 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.02654431626624354' -min_samples_leaf '57' -min_samples_split '65' -min_weight_fraction_leaf '0.08930549150722683' -n_estimators '379'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:58_177182_1192251119.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:58_177182_1192251119.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:58_177182_1192251119.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "8", "max_depth_type": "int", "max_features_float": "0.5726148176369666", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.23332877172857824", "min_samples_leaf": "55", "min_samples_split": "83", "min_weight_fraction_leaf": "0.36348637961614927", "n_estimators": "377", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "5859081", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 5859081 -bootstrap 'False' -criterion 'gini' -max_depth '8' -max_depth_type 'int' -max_features_float '0.5726148176369666' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.23332877172857824' -min_samples_leaf '55' -min_samples_split '83' -min_weight_fraction_leaf '0.36348637961614927' -n_estimators '377'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:13_177510_3072717526.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:13_177510_3072717526.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:13_177510_3072717526.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth": "60", "max_depth_type": "int", "max_features_float": "0.7312180988139318", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples": "0.41306657934026875", "max_samples_type": "float", "min_impurity_decrease": "0.29471144955868234", "min_samples_leaf": "100", "min_samples_split": "66", "min_weight_fraction_leaf": "0.44974166074778266", "n_estimators": "287", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3748112", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3748112 -bootstrap 'True' -criterion 'gini' -max_depth '60' -max_depth_type 'int' -max_features_float '0.7312180988139318' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples '0.41306657934026875' -max_samples_type 'float' -min_impurity_decrease '0.29471144955868234' -min_samples_leaf '100' -min_samples_split '66' -min_weight_fraction_leaf '0.44974166074778266' -n_estimators '287' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:28_177763_1802569077.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:28_177763_1802569077.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:28_177763_1802569077.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth": "62", "max_depth_type": "int", "max_features_float": "0.8713711553267762", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.023866763747827424", "min_samples_leaf": "7", "min_samples_split": "77", "min_weight_fraction_leaf": "0.4030964328653762", "n_estimators": "182", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "746784", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 746784 -bootstrap 'True' -criterion 'log_loss' -max_depth '62' -max_depth_type 'int' -max_features_float '0.8713711553267762' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.023866763747827424' -min_samples_leaf '7' -min_samples_split '77' -min_weight_fraction_leaf '0.4030964328653762' -n_estimators '182' -oob_score 'True'  
[INFO ] Incumbent changed to: config 8 (internal ID: 0x160F9), with mean quality: -0.9708333333333333; estimate based on 8 runs.
[INFO ] Sample call for new incumbent config 8 (internal ID: 0x160F9): 
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 15870886 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.18547141561103223' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:51_179046_2855764590.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:51_179046_2855764590.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:51_179046_2855764590.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.032409816085083754", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.4467285080830721", "min_samples_leaf": "95", "min_samples_split": "70", "min_weight_fraction_leaf": "0.09307478567156169", "n_estimators": "366", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "746784", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 746784 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.032409816085083754' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.4467285080830721' -min_samples_leaf '95' -min_samples_split '70' -min_weight_fraction_leaf '0.09307478567156169' -n_estimators '366'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.9633333333333334; estimate now based on 10 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:21_179636_4270597054.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:21_179636_4270597054.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:21_179636_4270597054.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "log2", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.27000113463370906", "min_samples_leaf": "62", "min_samples_split": "26", "min_weight_fraction_leaf": "0.3356022418265688", "n_estimators": "142", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3748112", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3748112 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'log2' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.27000113463370906' -min_samples_leaf '62' -min_samples_split '26' -min_weight_fraction_leaf '0.3356022418265688' -n_estimators '142'  
[INFO ] Incumbent changed to: config 12 (internal ID: 0x24EB8), with mean quality: -0.9638888888888889; estimate based on 12 runs.
[INFO ] Sample call for new incumbent config 12 (internal ID: 0x24EB8): 
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 15870886 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '14' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:13_181449_3560746012.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:13_181449_3560746012.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:13_181449_3560746012.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth": "67", "max_depth_type": "int", "max_features_float": "0.45262759692491317", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples": "0.7540409779683909", "max_samples_type": "float", "min_impurity_decrease": "0.46008387214844076", "min_samples_leaf": "34", "min_samples_split": "67", "min_weight_fraction_leaf": "0.3344845060481458", "n_estimators": "302", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3982386", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3982386 -bootstrap 'True' -criterion 'entropy' -max_depth '67' -max_depth_type 'int' -max_features_float '0.45262759692491317' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples '0.7540409779683909' -max_samples_type 'float' -min_impurity_decrease '0.46008387214844076' -min_samples_leaf '34' -min_samples_split '67' -min_weight_fraction_leaf '0.3344845060481458' -n_estimators '302' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:43_181866_3278881906.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:43_181866_3278881906.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:43_181866_3278881906.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth": "71", "max_depth_type": "int", "max_features_float": "0.18688722989167728", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples": "0.7392200436214637", "max_samples_type": "float", "min_impurity_decrease": "0.34452286880728317", "min_samples_leaf": "27", "min_samples_split": "7", "min_weight_fraction_leaf": "0.3887958962294619", "n_estimators": "81", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "613434", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 613434 -bootstrap 'True' -criterion 'entropy' -max_depth '71' -max_depth_type 'int' -max_features_float '0.18688722989167728' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples '0.7392200436214637' -max_samples_type 'float' -min_impurity_decrease '0.34452286880728317' -min_samples_leaf '27' -min_samples_split '7' -min_weight_fraction_leaf '0.3887958962294619' -n_estimators '81' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:57_182075_2670345362.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:57_182075_2670345362.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:57_182075_2670345362.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.4887659060456556", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "3.8626019684773194E-4", "min_samples_leaf": "2", "min_samples_split": "12", "min_weight_fraction_leaf": "0.43728065482315787", "n_estimators": "474", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "746784", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 746784 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.4887659060456556' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '3.8626019684773194E-4' -min_samples_leaf '2' -min_samples_split '12' -min_weight_fraction_leaf '0.43728065482315787' -n_estimators '474'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:27_182645_614498077.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:27_182645_614498077.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:27_182645_614498077.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.0", "min_samples_leaf": "1", "min_samples_split": "2", "min_weight_fraction_leaf": "0.0", "n_estimators": "100", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3748112", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3748112 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:42_182843_714271152.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:42_182843_714271152.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:42_182843_714271152.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth": "5", "max_depth_type": "int", "max_features_float": "0.19931495367812613", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.18501984357723983", "min_samples_leaf": "48", "min_samples_split": "6", "min_weight_fraction_leaf": "0.3340844863045139", "n_estimators": "120", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "16286516", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 16286516 -bootstrap 'False' -criterion 'entropy' -max_depth '5' -max_depth_type 'int' -max_features_float '0.19931495367812613' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.18501984357723983' -min_samples_leaf '48' -min_samples_split '6' -min_weight_fraction_leaf '0.3340844863045139' -n_estimators '120'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.9583333333333334; estimate now based on 20 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:57_183056_1809310789.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:57_183056_1809310789.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:57_183056_1809310789.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.5", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.0", "min_samples_leaf": "1", "min_samples_split": "2", "min_weight_fraction_leaf": "0.0", "n_estimators": "100", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "15870886", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 15870886 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.5' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:42_183810_2048822142.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:42_183810_2048822142.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:42_183810_2048822142.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes": "987", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.2508465348753413", "min_samples_leaf": "50", "min_samples_split": "61", "min_weight_fraction_leaf": "0.0478645704911736", "n_estimators": "273", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "746784", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 746784 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes '987' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.2508465348753413' -min_samples_leaf '50' -min_samples_split '61' -min_weight_fraction_leaf '0.0478645704911736' -n_estimators '273'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:12_184391_3670743952.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:12_184391_3670743952.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:12_184391_3670743952.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth": "99", "max_depth_type": "int", "max_features_special": "None", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.32771856947812317", "min_samples_leaf": "31", "min_samples_split": "99", "min_weight_fraction_leaf": "0.4344101583522928", "n_estimators": "226", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "746784", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 746784 -bootstrap 'False' -criterion 'entropy' -max_depth '99' -max_depth_type 'int' -max_features_special 'None' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.32771856947812317' -min_samples_leaf '31' -min_samples_split '99' -min_weight_fraction_leaf '0.4344101583522928' -n_estimators '226'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:42_184826_1734905693.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:42_184826_1734905693.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:42_184826_1734905693.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "80", "max_depth_type": "int", "max_features_float": "0.20452940192532032", "max_features_type": "float", "max_leaf_nodes": "281", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.1579216557164576", "min_samples_leaf": "35", "min_samples_split": "97", "min_weight_fraction_leaf": "0.1040167279703208", "n_estimators": "141", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "11733178", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 11733178 -bootstrap 'False' -criterion 'gini' -max_depth '80' -max_depth_type 'int' -max_features_float '0.20452940192532032' -max_features_type 'float' -max_leaf_nodes '281' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.1579216557164576' -min_samples_leaf '35' -min_samples_split '97' -min_weight_fraction_leaf '0.1040167279703208' -n_estimators '141'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:58_185036_2648676509.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:58_185036_2648676509.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:58_185036_2648676509.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth": "59", "max_depth_type": "int", "max_features_special": "None", "max_features_type": "special", "max_leaf_nodes": "64", "max_leaf_nodes_type": "int", "max_samples_type": "None", "min_impurity_decrease": "0.061933812524261056", "min_samples_leaf": "2", "min_samples_split": "1", "min_weight_fraction_leaf": "0.3860442744278507", "n_estimators": "318", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "4735669", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 4735669 -bootstrap 'True' -criterion 'gini' -max_depth '59' -max_depth_type 'int' -max_features_special 'None' -max_features_type 'special' -max_leaf_nodes '64' -max_leaf_nodes_type 'int' -max_samples_type 'None' -min_impurity_decrease '0.061933812524261056' -min_samples_leaf '2' -min_samples_split '1' -min_weight_fraction_leaf '0.3860442744278507' -n_estimators '318' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:12_185365_1494154151.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:12_185365_1494154151.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:12_185365_1494154151.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.9638561862388408", "max_features_type": "float", "max_leaf_nodes": "654", "max_leaf_nodes_type": "int", "max_samples": "0.3764840106249015", "max_samples_type": "float", "min_impurity_decrease": "0.46551972447914675", "min_samples_leaf": "45", "min_samples_split": "33", "min_weight_fraction_leaf": "0.07195672820674104", "n_estimators": "215", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "6672055", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 6672055 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.9638561862388408' -max_features_type 'float' -max_leaf_nodes '654' -max_leaf_nodes_type 'int' -max_samples '0.3764840106249015' -max_samples_type 'float' -min_impurity_decrease '0.46551972447914675' -min_samples_leaf '45' -min_samples_split '33' -min_weight_fraction_leaf '0.07195672820674104' -n_estimators '215' -oob_score 'False'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.9577777777777778; estimate now based on 30 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:43_185786_492983922.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:43_185786_492983922.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:43_185786_492983922.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth_type": "None", "max_features_float": "0.8018222462108859", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.24068228865799957", "min_samples_leaf": "43", "min_samples_split": "57", "min_weight_fraction_leaf": "0.2057536617099111", "n_estimators": "3", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "491360", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 491360 -bootstrap 'False' -criterion 'entropy' -max_depth_type 'None' -max_features_float '0.8018222462108859' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.24068228865799957' -min_samples_leaf '43' -min_samples_split '57' -min_weight_fraction_leaf '0.2057536617099111' -n_estimators '3'  
[INFO ] 
=======================================================================================
SMAC has finished. Reason: wall-clock time limit (603.478 s) has been reached.   
Total number of runs performed: 80, total configurations tried: 31.
Total CPU time used: 64 s, total wallclock time used: 603 s.
SMAC's final incumbent: config 12 (internal ID: 0x24EB8), with estimated mean quality: -0.9583333333333334, based on 32 run(s) on 1 training instance(s).
Sample call for this final incumbent:
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 491360 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '14' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'
=======================================================================================
[INFO ] 
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Additional information about run 23 in:/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Organising done! See Output/Performance_Data/performance_data.csv for results.
